src/elvis.erl

```diff
@@ -57,13 +57,15 @@ option_spec_list() ->
     OutputFormat = "It allows you to display the results in plain text. When "
                    ++ "none is provided elvis displays the results in colors. "
                    ++ "The options allowed are (plain | colors).",
+    KeepGoing = "Won't stop rocking on first error when given a list of files",
     [
      {help, $h, "help", undefined, "Show this help information."},
      {config, $c, "config", string, Commands},
      {commands, undefined, "commands", undefined, "Show available commands."},
      {output_format, undefined, "output-format", string, OutputFormat},
      {version, $v, "version", undefined, "Specify the elvis current version."},
-     {code_path, $p, "code-path", string, "Add the directory in the code path."}
+     {code_path, $p, "code-path", string, "Add the directory in the code path."},
```

Comment:
Line 67 is too long:      {code_path, $p, "code-path", string, "Add the directory in the code path."},.